lib/fog/ovirt/requests/compute/attach_volume.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Ovirt
+      class Real
+        def attach_volume(id, options)
+          raise ArgumentError, "instance id is a required parameter" unless id
+          raise ArgumentError, "volume id is a required parameter for attach-volume" unless options.key? :id
+
+          client.attach_volume(id, options[:id])
```

Comment:
I agree, the reason I did this is that all `*_volume` methods use same arguments. For instance [`destroy_volume`](https://github.com/fog/fog/blob/master/lib/fog/ovirt/requests/compute/destroy_volume.rb) would only need a `volume_id` to be removed.
Should I update?